gitlab/cli.py

```diff
@@ -299,11 +300,15 @@ def _parse_value(v: Any) -> Any:
     if isinstance(v, str) and v.startswith("@"):
         # If the user-provided value starts with @, we try to read the file
         # path provided after @ as the real value. Exit on any error.
+        filepath = pathlib.Path(v[1:]).expanduser().resolve()
         try:
-            with open(v[1:], encoding="utf-8") as f:
+            with open(filepath, encoding="utf-8") as f:
                 return f.read()
-        except Exception as e:
-            sys.stderr.write(f"{e}\n")
+        except UnicodeDecodeError:
+            with open(filepath, "rb") as f:
+                return f.read()
+        except (FileNotFoundError, PermissionError) as exc:
```

Comment:
Maybe we should also add `IsADirectoryError` or use the parent OSError here?

Reply:
Done using `OSError`